broker-cli/commands/market/market-stats.js

```diff
@@ -6,8 +6,8 @@ const { handleError } = require('../../utils')
  * ex: `sparkswap market market-stats --market btc/ltc'
  *
  * @param {Object} opts
- * @param {String} opts.market
- * @param {String} [rpcaddress] opts.rpcaddress
+ * @param {string} opts.market
+ * @param {string} [rpcaddress] - opts.rpcaddress
```

Comment:
this seems wrong, isn't the thing after the dash the description?

Reply:
@treygriffith yes. This didnt convert correctly because `[rpcaddress]` is incorrect. it should be `[opts.rpcaddress=nil]` and then description is after it